ydb/public/lib/ydb_cli/commands/ydb_profile.cpp

```diff
@@ -1043,11 +1043,12 @@ int TCommandUpdateProfile::Run(TConfig& config) {
 }
 
 TCommandReplaceProfile::TCommandReplaceProfile()
-            : TCommandProfileCommon("replace", {}, "Create new configuration profile or delete and re-configure existing one")
+            : TCommandProfileCommon("replace", {}, "Deletes profile and creates a new with the same name")
```

Comment:
`Deletes profile and creates a new one with the same name and new property values from provided options. If no options are provided, a profile will be replaced with an empty one`
Not sure about the last sentence so its up to you